riot/riot.py

```diff
@@ -165,7 +168,10 @@ def path(self) -> str:
     def venv_path(self) -> str:
         """Return the path to the virtual environment for this interpreter."""
         version = self.version().replace(".", "")
-        return os.path.abspath(f".riot/venv_py{version}")
+        env_base_path = os.environ.get("RIOT_ENV_BASE_PATH") or DEFAULT_RIOT_PATH
```

Comment:
```suggestion
        env_base_path = os.environ.get("RIOT_ENV_BASE_PATH", DEFAULT_RIOT_PATH)
```
Pretty sure you can just specify the default like this.